### PR TITLE
adds Stylus by Example pages to the docs

### PR DESCRIPTION
**Pull request by douglance**

### CODE REVIEW

**Review by TucksonDev — APPROVED** [review]
LGTM, thanks very much @douglance !

File: website/sidebars.js
Comment (anegg0) [inline website/sidebars.js:1]: For now, I would place the link to SBE under "Rust crate docs" to avoid duplicates.
We'll figure out a better-suited area in the upcoming iteration.

**Review by anegg0 — CHANGES_REQUESTED** [review]
Looking great to me.
I added a suggestion regarding SBE's link placement.
Will merge right after the change.

**Review by anegg0 — APPROVED** [review]
Awesome! Thanks much, @douglance!

**Pull request merged**
